homeassistant/components/cloud/http_api.py

```diff
@@ -436,7 +445,7 @@ async def websocket_subscription(
         vol.Optional(PREF_GOOGLE_REPORT_STATE): bool,
         vol.Optional(PREF_GOOGLE_SECURE_DEVICES_PIN): vol.Any(None, str),
         vol.Optional(PREF_TTS_DEFAULT_VOICE): vol.All(
-            vol.Coerce(tuple), vol.In(MAP_VOICE)
+            vol.Coerce(tuple), validate_language_voice
```

Comment:
Since we're anyway changing the corresponding parameter, should we make this accept a dict with keys "language" and "voice" instead? `TTS_VOICES` is a dict of dicts, so I don't think we need to keep this a tuple?

Reply:
I like that the choice is an item in the container we return in the `tts/info` api. Changing to something that doesn't use tuples will make that hard. I don't have any further opinions about this.

Reply:
Yeah, but frontend will need to either reshape the response from `tts/info` to a map, or search for the voices for a certain language, so just returning `TTS_VOICES` as is would be more natural.

Anyhow, as discussed on Discord, we agree to disagree on this one so let's leave it at that.